### ISSUE TITLE
Add Binding to Reset Zoom for Ctrl + NumPad0

### ISSUE THREAD

**Issue by joshualyon** [ShareX/ShareX#6396]
https://github.com/ShareX/ShareX/blob/6dec8a9a10a379fe3ae557813592bbfc252d97b8/ShareX.ScreenCaptureLib/Forms/RegionCaptureForm.cs#L669

TLDR: Request to add keybinding for `Keys.NumPad0` in addition to `Keys.D0` to reset zoom.

The Zoom additions in 14.x are great! The changelog mentioned there was a keybinding for <kbd>Ctrl</kbd>+<kbd>0</kbd> to reset the zoom and at first I thought it wasn't working as I kept trying to use the `0` key from the num pad. Looking at the source code, it looks like the binding is explicitly for `Keys.D0` which is for the number keys at the top of the keyboard. There [appears to be](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.keys?view=windowsdesktop-6.0) a separate `Keys.NumPad0` key that would need to be bound to cover the num pad on the side of a full size keyboard.

**Comment by Jaex** [comment]
Added in #6334
